pkgs/by-name/un/unifont/package.nix

```diff
@@ -20,6 +20,11 @@ stdenv.mkDerivation rec {
     hash = "sha256-AqP+EZlNPNrx1L1dK2tglzXmgj4BdkroO3BOAuwvZA0=";
   };
 
+  bdf = fetchurl {
+    url = "mirror://gnu/unifont/${pname}-${version}/${pname}-${version}.bdf.gz";
```

Comment:
```suggestion
    url = "mirror://gnu/unifont/unifont-${version}/unifont-${version}.bdf.gz";
```

Reply:
Would you like to change the other two as well?

Reply:
Yep. We don't want to use pname like this anymore because it does not interact nicely with finalAttrs